src/App.tsx

```diff
@@ -9,39 +9,25 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
-import { User } from './types/User';
-import { Post } from './types/Post';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { selectedPostSLice } from './features/selectedPost/selectedPostSlice';
+import { postSLice, setPosts } from './features/posts/postsSlice';
```

Comment:
The import `postSLice` should be renamed to `postSlice` to follow JavaScript naming conventions and match the export from `postsSlice.ts` .